src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureProvider.java

```diff
@@ -88,15 +91,18 @@
  */
 @Component
 @Log4j2
-public abstract class AzureProvider implements CloudProvider {
+public class AzureProvider implements CloudProvider {
 
     @Getter
-    protected final String type = "azureAbstract";
+    protected final String type = "azure";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)